cli/src/utils/renderVersionWarning.ts

```diff
@@ -1,6 +1,14 @@
-import { execSync } from "child_process";
 import { getVersion } from "./getT3Version.js";
 import { logger } from "./logger.js";
+import { execSync } from "child_process";
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the LICENSE file in the root
+ * directory of this source tree.
+ * https://github.com/facebook/create-react-app/blob/main/packages/create-react-app/LICENSE
+ */
+import https from "https";
```

Comment:
i wish import sorters didn't move imports when they are separated by actual code blocks...

is there a setting for this?

Reply:
Looks like you can `//prettier-ignore` lines but thats about it :/